### PR TITLE
Improvements to the Reactive SQL Clients guide

### LINKED ISSUE #39601: Guide on Reactive SQL incomplete
### Describe the bug

the [reactive-sql-clients guide](https://quarkus.io/guides/reactive-sql-clients) mentions creating an html file fruits.html but it doesn't mention where to place this file (and also where to place fruits.js)
Moreover I think it's also missing an additional endpoint to FruitResource to serve this html file from http://localhost:8080/fruits.html

I fixed it by adding it under src/main/resources and adding this endpoint:
```java
  @GET
  @Path("/fruits.html")
  @Produces(MediaType.TEXT_HTML)
  public Response getHtml() {
    try {
      String html = new String(Files.readAllBytes(Paths.get(getClass().getClassLoader().getResource("fruits.html").toURI())));
      return Response.ok(html).build();
    } catch (Exception e) {
      return Response.serverError().entity("Error loading fruits.html").build();
    }
  }
```

### Expected behavior

Guide should be complete

### Actual behavior

Guide doesn't include all the necessary code and information 

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tsegismont**
Fixes #39601

Also:

- adds a link to the quickstart
- informs the reader about dev services for databases
- updates snippets

### CODE REVIEW

File: docs/src/main/asciidoc/reactive-sql-clients.adoc [hunk 1]
```diff
@@ -31,21 +31,34 @@ Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing
 
 In this guide, you will learn how to implement a simple CRUD application exposing data stored in *PostgreSQL* over a RESTful API.
 
+The code presented is available in this {quickstarts-base-url}[GitHub repository] under the link:{quickstarts-tree-url}/getting-started-reactive-crud[`getting-started-reactive-crud` directory].
```
Comment (gsmet) [inline docs/src/main/asciidoc/reactive-sql-clients.adoc:34]: Could you mimick what we do in the other guides instead?

Typically https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/validation.adoc?plain=1#L30-L37 .

We try to have the same structure for all guides.

Now sometimes it doesn't make sense so if you think it doesn't make sense for you, please push back.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/reactive-sql-clients.adoc (hunk 1 above)
Comment (tsegismont) [reply docs/src/main/asciidoc/reactive-sql-clients.adoc:34]: Will do, thanks for the feedback

Comment (tsegismont) [reply docs/src/main/asciidoc/reactive-sql-clients.adoc]: Done

File: docs/src/main/asciidoc/reactive-sql-clients.adoc [hunk 2]
```diff
@@ -59,16 +70,32 @@ public class Fruit {
 }
 ----
 
+== Prerequisites
+
+{prerequisites-docker}
```
Comment (gsmet) [inline docs/src/main/asciidoc/reactive-sql-clients.adoc:75]:
```suggestion
:prerequisites-docker:
```

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/reactive-sql-clients.adoc (hunk 2 above)
Comment (gsmet) [reply docs/src/main/asciidoc/reactive-sql-clients.adoc:75]: (What you used displays the attribute, what I suggest sets its value)

Comment (tsegismont) [reply docs/src/main/asciidoc/reactive-sql-clients.adoc]: Sorry for that, and thank for reviewing

**Review by gsmet — APPROVED** [review]
No worries, thanks for the adjustment!

**Pull request merged**